src/alire/alire-crate_configuration.adb

```diff
@@ -25,13 +25,18 @@ with TOML; use TOML;
 
 package body Alire.Crate_Configuration is
 
+   Separator : constant Character := ',';
+   Assign    : constant Character := ':';
```

Comment:
Why not `=` for assign?

Reply:
I was in a mindset of mappings, JSON-style, and it seemed a bit less ugly than `--profiles=crate=profile`. I'm trying to remember if we have something similar in another place to follow convention. It's true that TOML uses = for maps...